runtime/calamari/src/lib.rs

```diff
@@ -505,13 +701,24 @@ construct_runtime!(
 		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 10,
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage} = 11,
 
+		// Governance stuff.
+		Democracy: pallet_democracy::{Pallet, Call, Storage, Config<T>, Event<T>} = 14,
+		Council: pallet_collective::<Instance1>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 15,
+		TechnicalCommittee: pallet_collective::<Instance2>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 16,
+		//PhragmenElection: pallet_elections_phragmen::{Pallet, Call, Storage, Event<T>, Config<T>} = 17,
+		//TechnicalMembership: pallet_membership::<Instance1>::{Pallet, Call, Storage, Event<T>, Config<T>} = 18,
```

Comment:
TODO: Still need to figure out what the `pallet_elections_phragmen` and `pallet_membership` are used for.
Moonriver hasn't implemented them like this runtime.
But Acala/Khala/Kusama use them.

Reply:
`pallet_elections_phragmen` is used to enable elections for the `pallet_collective` instances (Council and Tech Committee)
Independent of `pallet_membership`
Only Khala and Kusama implement it now.